project-base/app/config/cron.yaml

```diff
@@ -102,3 +102,9 @@ services:
     Shopsys\FrameworkBundle\Component\Packetery\PacketeryCronModule:
         tags:
             - { name: shopsys.cron, hours: '*', minutes: '*/5', instanceName: packetery, readableName: 'Sending parcels to the packetery' }
+
+    # Watchdog
+
+    Shopsys\FrameworkBundle\Model\Watchdog\WatchdogCronModule:
```

Comment:
new instance should be also added to `docker/conf/docker-compose.github-actions.review.simplified.yml.dist`, `docker/conf/docker-compose.github-actions.review.yml.dist` to the section with chadburn entries (running cron on CI).